File: 35_최현진/로또의 최고 순위와 최저 순위.py
def solution(lottos, win_nums):
    answer = []
    cnt_collect = 0
    cnt_0 = lottos.count(0)
    rank = [6, 6, 5, 4, 3, 2, 1]

    for i in lottos:
        if i in win_nums:
            cnt_collect += 1

    answer.append(rank[cnt_0 + cnt_collect])
    answer.append(rank[cnt_collect])

    return answer

File: 27_이유리/2주차/로또의최고순위와최저순위.java
class Solution {
    public int[] solution(int[] lottos, int[] win_nums) {
        int[] answer = {};
        int count = 0;	// 당첨이 확실한 번호의 수
        int zero = 0;	// 지워진 번호의 수
        
        for(int i = 0; i < 6; i++) {
        	if(lottos[i] == 0) {
        		zero++;
        		continue;
        	}
        	for(int j = 0; j < 6; j++) {
        		if(lottos[i] == win_nums[j])
        			count++;
        	}
        }
        
        if(count == 0) {
        	if(zero == 0)
        		answer = new int[] {6, 6};
        	else
        		answer = new int[]{ 7 - zero, 6};
        }
        else {
        	answer = new int[]{ 7 - (count + zero), 7 - count};
        }
        
        return answer;
    }
}
